alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
+
+# Configure nix
+COPY config.nix /root/.nixpkgs/config.nix
+
+# Install system dependencies
+RUN nix-env --install --fallback \
+    acl-2.2.52 \
+    bundler-1.10.5 \
```

Comment:
The current release of bundler is `bundler-1.10.6`, I guess we should upgrade the reference.